src/Helper/Helper_Notices.php

```diff
@@ -225,4 +228,38 @@ private function html( $text, $class = 'updated' ) {
 
 		remove_filter( 'wp_kses_allowed_html', $allow_form_elements );
 	}
+
+	/**
+	 * Return Gravity PDF error messages and disregard the existing messages.
+	 *
+	 * @param array $messages The message to be displayed
+	 *
+	 * @return array $this->errors
+	 *
+	 * @since 6.5
+	 */
+	public function override_gravityforms_admin_error_messages( $messages ): array {
+		if ( ! $this->has_error() ) {
+			return [];
+		}
+
+		return $this->errors;
+	}
+
+	/**
+	 * Return Gravity PDF notice messages and disregard the existing messages.
+	 *
+	 * @param array $messages The message to be displayed
+	 *
+	 * @return array $this->notices
+	 *
+	 * @since 6.5
+	 */
+	public function override_gravityforms_admin_messages( $messages ): array {
+		if ( ! $this->has_notice() ) {
+			return [];
+		}
+		return $this->notices;
```

Comment:
@jestonihpi for consistency, add a new line before the return statement, like in the method above.